.github/workflows/R_CMD_check_Hades.yaml

```diff
@@ -39,6 +39,12 @@ jobs:
 
       - uses: r-lib/actions/setup-pandoc@v2
 
+      - name: Install DatabaseConnector for tests
```

Comment:
Dependencies in Suggests are automatically installed by the default script, so this is not necessary.

Reply:
I was getting an error on github actions and adding that line fixed it. I can try removing it. https://github.com/OHDSI/Eunomia/actions/runs/5281622666/jobs/9555457012#step:12:90

<img width="597" alt="image" src="https://github.com/OHDSI/Eunomia/assets/10227522/d5023ddf-aaba-4112-ae9e-bdc78c6784a6">